src/go/github-runner-monitor/pkg/machines/machine.go

```diff
@@ -30,6 +30,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var timeout = 30
```

Comment:
```suggestion
var timeout = 30 * time.Second
```

That makes it clearer what unit this is in (and fixes the cast below).

Also, shouldn't this be a `const`?